pkg/services/playlist/model.go

```diff
@@ -13,11 +13,11 @@ var (
 
 // Playlist model
 type Playlist struct {
-	Id       int64  `json:"id"`
-	UID      string `json:"uid" xorm:"uid"`
-	Name     string `json:"name"`
-	Interval string `json:"interval"`
-	OrgId    int64  `json:"-"`
+	Id       int64  `json:"id,omitempty" db:"id"`
```

Comment:
`ID` for the go model I think, same with `OrgID`

Reply:
Indeed, there are other things we need to refectory still, I just touched what related to sqlx, will do a refectory after each xorm -> sqlx, our storage is not clean.

Reply:
It is okay, I thought the model was being moved when I first looked at this - sorry for confusion.